### PR TITLE
#58 register i18n translation tables

### LINKED ISSUE #58: [i18n] inject translations
Details [here](https://ci.linagora.com/linagora/lgs/openpaas/esn/-/wikis/Frontend-Split/Polish/i18n#code-changes-howto-solution-1-kiss)

### PR DESCRIPTION
**Pull request by tuanlc**

### CODE REVIEW

File: src/modules/linagora.esn.account/frontend/js/app.run.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ require('./services.js');
   angular.module('linagora.esn.account')
   .run(run)
   .run(addTemplateCache);
-
+
```
Comment (itswillta) [inline src/modules/linagora.esn.account/frontend/js/app.run.js]: One redundant empty space.

File: src/modules/linagora.esn.account/frontend/js/app.run.js [hunk 2]
```diff
@@ -21,29 +21,29 @@ require('./services.js');
     accountService.getAccountProviders()
       .then(function(providers) {
         var supportedProviders = _.intersection(providers.data, _.values(SUPPORTED_ACCOUNT_TYPES));
-
+
```
Comment (itswillta) [inline src/modules/linagora.esn.account/frontend/js/app.run.js]: One redundant empty space.

File: src/modules/linagora.esn.account/frontend/js/app.run.js [hunk 3]
```diff
@@ -21,29 +21,29 @@ require('./services.js');
...
+  
         supportedProviders.forEach(function(provider) {
           var options = {
             attributes: [{
               name: 'type',
               value: provider
             }]
           };
-
+
```
Comment (itswillta) [inline src/modules/linagora.esn.account/frontend/js/app.run.js]: One redundant empty space.

File: src/modules/linagora.esn.account/frontend/js/app.run.js [hunk 4]
```diff
@@ -21,29 +21,29 @@ require('./services.js');
...
             }]
           };
-
+  
           var directive = new dynamicDirectiveService.DynamicDirective(
             function() {
               return true;
             }, 'account-menu-item', options);
-
+
```
Comment (itswillta) [inline src/modules/linagora.esn.account/frontend/js/app.run.js]: One redundant empty space.

File: src/modules/linagora.esn.account/frontend/js/app.run.js [hunk 5]
```diff
@@ -21,29 +21,29 @@ require('./services.js');
...
               return true;
             }, 'account-menu-item', options);
-
+  
           dynamicDirectiveService.addInjection(FAB_ANCHOR_POINT, directive);
           accountMessageRegistry.register(provider, socialHelper.getAccountMessages(provider));
         });
       });
-
+
```
Comment (itswillta) [inline src/modules/linagora.esn.account/frontend/js/app.run.js]: One redundant empty space.

File: src/modules/linagora.esn.account/frontend/js/app.run.js [hunk 6]
```diff
@@ -21,29 +21,29 @@ require('./services.js');
...
           accountMessageRegistry.register(provider, socialHelper.getAccountMessages(provider));
         });
       });
-
+  
     var accountControlCenterMenu = new dynamicDirectiveService.DynamicDirective(true, 'controlcenter-menu-account', {priority: -2});
     dynamicDirectiveService.addInjection('controlcenter-sidebar-menu', accountControlCenterMenu);
   }
-
+
```
Comment (itswillta) [inline src/modules/linagora.esn.account/frontend/js/app.run.js]: One redundant empty space.

File: src/modules/linagora.esn.account/frontend/js/app.run.js (hunk 1 above)
Comment (piavgh) [inline src/modules/linagora.esn.account/frontend/js/app.run.js]: Please delete the spaces here and below

**Review by itswillta — APPROVED** [review]

**Review by piavgh — APPROVED** [review]

**Pull request merged**